docs/index.asciidoc

```diff
@@ -58,6 +58,14 @@ input {
 
 |=======================================================================================================================
 
+.Technology Preview
+****
+This {esf-name} input plugin is part of a _Technology Preview_, which means that both configuration options and implementation details are subject to change in minor releases without being preceded by deprecation warnings.
+
+Before upgrading this plugin or Logstash itself, please pay special attention to the changelogs to avoid being caught by surprise.
```

Comment:
Can we be more specific here?  Do we mean the changelog for this plugin?  
If so, please consider making "changelog" singular or even specifying "CHANGELOG.md" and including a link.